components/classes/StudentListTableBody.tsx

```diff
@@ -4,6 +4,7 @@ import { Student } from "@/utils/types/person";
 import { list } from "radash";
 import { MAXIMUM_EMPTY_COLUMNS, OptionsType } from "@/components/classes/StudentListPrintout";
 import { StylableFC } from "@/utils/types/common";
+import permitted from "@/utils/helpers/permitted";
```

Comment:
Where is this used?

Reply:
I originally thought this had to be used while editing, and didn't remove before the push. I'll remove that right away.